src/psbt.h

```diff
@@ -27,11 +27,24 @@ static constexpr uint8_t PSBT_IN_WITNESSSCRIPT = 0x05;
 static constexpr uint8_t PSBT_IN_BIP32_DERIVATION = 0x06;
 static constexpr uint8_t PSBT_IN_SCRIPTSIG = 0x07;
 static constexpr uint8_t PSBT_IN_SCRIPTWITNESS = 0x08;
+static constexpr uint8_t PSBT_IN_VALUE = 0x09;
```

Comment:
Let's move these values to somewhere vendor-specific

Reply:
Right, sounds like we're moving them up to start at 0xF0.

Reply:
Done.